llama/ssm.py

```diff
@@ -1,12 +1,18 @@
+import os
+
 from boto3 import client
 
 
 class SSM:
     """An SSM class that provides a generic boto3 SSM client with specific SSM
     functionality necessary for llama scripts"""
 
-    def __init__(self):
-        self.client = client("ssm", region_name="us-east-1")
+    def __init__(self, endpoint_url=None):
```

Comment:
We don't need to pass `endpoint_url` as an arg here, as it isn't actually used in the client instantiation below.